apps/src/applab/designElements/RestoreThemeDefaultsButton.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import commonStyles from '../../commonStyles';
+import PropTypes from 'prop-types';
+import Radium from 'radium';
+
+const styles = {
+  restoreButton: {
+    backgroundColor: '#0aa',
+    color: 'white',
+    float: 'right'
+  }
+};
+
+/**
+ * A restore theme defaults button
+ */
+class RestoreThemeDefaultsButton extends React.Component {
+  static propTypes = {
+    handleRestore: PropTypes.func.isRequired
+  };
+
+  render() {
+    const {handleRestore} = this.props;
+    return (
+      <button
+        type="button"
+        style={[commonStyles.button, styles.restoreButton]}
+        onClick={handleRestore}
```

Comment:
Should we add a verification message/dialog/button?
@ryansloan

Reply:
Hmm, good question. I don't think I'd add a verification step here since at this point you may need to do it for a lot of controls, and it's not exactly _destructive_ if you do it accidentally (you keep your control, the style for that one control just changes). I don't have a ton of (non-)competitive products to research on this, but I played with some word processors and spreadsheets and generally the "clear formatting" step doesn't have a confirmation (though they do have "undo").

I think we should release it without a confirmation step and see if we hear of or notice any issues.

Aside: In pondering this, I noticed the **delete** button doesn't have a confirmation(!), which we may want to change in another PR.

Reply:
Sounds good.

Aside: Delete _screen_ has a confirmation, but not any of the other elements.

Reply:
Given unlimited resources, would we rather have "Undo/Redo"?

Reply:
Rather than this button? Given unlimited resources I would want to have both. Their use can overlap but but they also serve distinctly different purposes. If you look at most mainstream WYSIWYG editors (Excel/Sheets, Word/Docs, etc.) you will generally find both "undo/redo" and "clear formatting".